### ISSUE TITLE
RoundRectDataList: can't change data

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1553]
@jhpedemonte 

I can't change data using SmartInput on a RoundRectDataList on either Mac/Chrome or Mac/Safari using macwayne2. Exception at EdgeToEdgeDataListInput.js:32:

```
textArea = dijit.byId("davinciIleb"),
```

**Comment by JonFerraiolo** [comment]
THIS SUGGESTION WAS WRONG: I'm pretty sure this can be fixed by replacing `davinciI1eb` with `ieb`

**Comment by JonFerraiolo** [comment]
The problem is that updateStore() in EdgeToEdgeDataListInput is getting called twice, the second time after the textarea widget has been destroyed, which causes local variable textArea to be undefined.

**Comment by JonFerraiolo** [comment]
I got things working by making these changes:
- In DataStoreBasedWidgetInput.js, I renamed `updateWidget:function()` to `_updateWidget:function()` (add an undercore)
- In same file, I added a null function `updateWidget:function(){}` (this ultimately gets called by the `hide()` function in SmartInput.js)
- In same file, within `onOK` I call `this._updateWidget:function()` instead of `this.updateWidget:function()` (add an undercore)

As a result, updateStore() is called only once, and called when inline edit hasn't been destroyed yet.

Also, would be good to add a `var` before the variable initializations at the top of updateStore().

However, I don't know this code and @billreed63 needs to look at this and make the fixes.
